sklearn/linear_model/_logistic.py

```diff
@@ -1140,11 +1162,15 @@ class LogisticRegression(LinearClassifierMixin,
         For the liblinear and lbfgs solvers set verbose to any positive
         number for verbosity.
 
+        |ResourcesParameter|
```

Comment:
I think sklearn might need also `DebugParameter` type as well. (We do not have those in D3M.)